resolwe_bio/docker_images/chipseq/Dockerfile

```diff
@@ -0,0 +1,72 @@
+FROM docker.io/resolwe/base:ubuntu-14.04
+
+LABEL MAINTAINER Resolwe-bio authors https://github.com/genialis/resolwe-bio
+
+RUN apt-get update && \
+    apt-get install -y software-properties-common python python-dev python-pip && \
+
+    add-apt-repository -y ppa:fkrull/deadsnakes && \
+    apt-get update && \
+
+    apt-get -y install --no-install-recommends \
+      wget \
+      # required for bamliquidator
+      samtools \
+      libbam-dev \
+      libhdf5-serial-dev \
+      libboost-dev \
+      libboost-timer-dev \
+      libgoogle-perftools-dev \
+      libtbb-dev \
+      python-numpy \
+      python-scipy \
+      python-pandas \
+      python-redis \
+      python-software-properties \
+      python-tables \
+      python-numexpr \
+      # required for ROSE2
+      r-base \
+      # required for MACS1.4
+      python2.6 \
+      && \
+
+    apt-get clean && \
+
+    echo "Installing bokeh..." && \
+    pip install bokeh==0.9.3 && \
+
+    echo "Installing ROSE2..." && \
+    pip install rose2==1.0.5 && \
+
+    echo "Installing bamplot..." && \
+    sudo pip install bamplot==0.9.0 && \
+
+    echo "Installing MACS2..." && \
+    pip install MACS2==2.1.1.20160309 && \
+
+    echo "Installing MACS..." && \
+    MACS_VERSION=1.4.2 && \
+    MACS_SHA1SUM=3c50d21cb252a9cc89d567654194e1c4725fdf5a && \
+    wget -q https://github.com/downloads/taoliu/MACS/macs_$MACS_VERSION.deb -O macs.deb && \
+    echo "$MACS_SHA1SUM *macs.deb" | sha1sum -c - && \
+    dpkg -i macs.deb && \
+    rm macs.deb && \
+
+    echo "Installing bamliquidator..." && \
+    BAMLIQUIDATOR_VERSION=v1.3 && \
+    BAMLIQUIDATOR_SHA1SUM=b9f032b71cc0231df60e18979d23605d7daa9e6d && \
+    wget -q https://github.com/BradnerLab/pipeline/archive/bamliquidator_$BAMLIQUIDATOR_VERSION.tar.gz -O /opt/bamliquidator.tar.gz && \
+    echo "$BAMLIQUIDATOR_SHA1SUM */opt/bamliquidator.tar.gz" | sha1sum -c - && \
+    cd /opt && \
```

Comment:
Also, instead of using `cd <something>` followed by `cd ..`, use `pushd <something>` and `popd`.